patches/server/0814-Overridable-YAML-configuration.patch

```diff
@@ -0,0 +1,285 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cubxity <cubxity@protonmail.com>
```

Comment:
Coauthor lynx as you took a major jab from his patch :p 

```
Co-authored-by: Bjarne Koll <lynxplay101@gmail.com>
```

Reply:
My bad, I coauthored them now.